### PR TITLE
adding new norms to FGSM

### PR DESCRIPTION
**Pull request by npapernot**
added L1 and L2 norms for FGSM in TF

credit to @ftramer for this

### CODE REVIEW

File: cleverhans/attacks.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import numpy as np
 
 
-def fgsm(x, predictions, eps, back='tf', clip_min=None, clip_max=None):
+def fgsm(x, predictions, eps, back='tf', clip_min=None, clip_max=None,
+         ord=np.inf):
```
Comment (goodfeli) [inline cleverhans/attacks.py]: should add `ord` to the docstring

**Comment by goodfeli** [comment]
I think it would be good to add a test of each value of `ord`. A simple enough test might pass on travis, since we don't know what it is making the mnist accuracy test get low accuracy.

**Comment by goodfeli** [comment]
I won't insist on changing the name / interface, but have you given some thought as to whether it makes sense to still call this `fgsm` when `ord` is anything other than `np.inf`? The "s" in `fgsm` is for "sign" and we take the sign only if we are using the max norm.

**Comment by npapernot** [comment]
Closing this given that this will be added through the new API

**Pull request closed without merging**